db/schema.rb

```diff
@@ -34,6 +34,18 @@
     t.integer "user_id"
   end
 
+  create_table "profiles", force: :cascade do |t|
```

Comment:
why was this added in this PR

Reply:
This I have no idea what happend, Before this PR i solved merge conflicts - maybe its from that?
Dont have any memory of seeing this actualy.